apps/emqx/src/emqx_schema.erl

```diff
@@ -640,6 +640,14 @@ fields("flapping_detect") ->
                     default => "5m",
                     desc => ?DESC(flapping_detect_ban_time)
                 }
+            )},
+        {"clean_when_banned",
```

Comment:
It should belong to the banned scope instead of flapping?

Reply:
but we have no banned config ...

Reply:
Maybe we should add one. :thinking:  Is it more clear concept structures? Whether clean banned client's retained/delayed messages is a concept that belongs to Banned

Reply:
don't think it is necessary, `flapping_detect` is an entry point for the banned and `clean` is an argument of the banned's API.
Banned only needs to provide an insert API with options which include a `clean` argument